### ISSUE TITLE
Dear Team, GSX announce a REST API change... would you please will change gsxlib?

### ISSUE THREAD

**Issue by alopezruiz** [filipp/gsxlib#70]
Dear @filipp et all, GSX announce a RESTful API change... 

Would you please will change gsxlib for this?

Thanks you!

**Comment by filipp** [comment]
Sure thing! I just got the email from Apple a couple of days ago. If I understood correctly, the migration will be done in two phases (June 2019 and early 2020) which presents an interesting challenge for the library - at some point we will have to support both the new and old APIs... either that or one would have to have two copies of gsxlib installed?

Is there a name or version number for the new GSX API version? Just wondering what I should call the git branch... 🤔

**Comment by tuxudo** [comment]
The current API is SOAP based right? Could the branch be named something like "RESTful"?

**Comment by CraigDSmith** [comment]
I'd name them something like GSX2019 and GSX2020

**Comment by gunwantsonawane** [comment]
@filipp Do you have any new lib for supporting GSX new REST APIs?

**Comment by CraigDSmith** [comment]
No. He's been taking a break for personal reasons. See this message:  https://github.com/filipp/gsxlib/issues/72

**Comment by upccjesse** [comment]
I started on my own GSX REST library in PHP at https://github.com/UniversityPCCare/gsx-rest-php if you'd like to check it out.

**Comment by Isaac-Duarte** [comment]
I have started a C# Library. https://github.com/Isaac-Duarte/Cosmos still under the works
